website/views/company.py

```
@@ -2110,3 +2108,297 @@ def check_domain_security_txt(request):
+    job = get_object_or_404(Job, pk=pk)
+
+    # Check if user is org member (admin or manager)
+    is_org_member = False
+    if request.user.is_authenticated:
+        is_org_member = (
+            job.organization.admin == request.user or job.organization.managers.filter(id=request.user.id).exists()
+        )
+
+    # Public users can only see active, public jobs
+    if not is_org_member and (not job.is_public or job.status != "active"):
+        raise Http404("Job not found")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)